template.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"io/ioutil"
+	"time"
+
+	"gopkg.in/yaml.v2"
+)
+
+type template struct {
+	Service string            `yaml:"service"`
+	Thrift  string            `yaml:"thrift"`
+	Method  string            `yaml:"method"`
```

Comment:
In YARPC, we are exposing method as "Procedure". This template could also expose "Caller" as an optional override, just as is possible at the CLI. "Baggage" is now an option on the dev branch as well. I think it will be good to punt on transport headers though.